namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-preprod/resources/rds.tf

```diff
@@ -0,0 +1,81 @@
+module "rds" {
+  source                       = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.16.13"
+  vpc_name                     = var.vpc_name
+  team_name                    = var.team_name
+  business-unit                = var.business_unit
+  application                  = var.application
+  is-production                = var.is_production
+  environment-name             = var.environment
+  infrastructure-support       = var.infrastructure_support
+  namespace                    = var.namespace
+  performance_insights_enabled = true
+  db_engine_version            = "13"
```

Comment:
do you mean to target the previous version of postgres here?

Reply:
followed you up on Slack, thanks for the prompt! https://mojdt.slack.com/archives/C69NWE339/p1669737852392029?thread_ts=1669732375.931909&cid=C69NWE339

Reply:
To close the loop, here's a [PR](https://github.com/ministryofjustice/cloud-platform-environments/pull/9349) we are thinking about merging in before this to prove that 14 is stable. If it is, I'll update this PR with 14 👍